contracts/protocol/facets/DisputeHandlerFacet.sol

```diff
@@ -37,7 +37,7 @@ contract DisputeHandlerFacet is DisputeBase, IBosonDisputeHandler {
      *
```

Comment:
Please add to reverts if section:
- The disputes region of protocol is paused

Reply:
done

Reply:
sorry for not being clearer, but it's missing from IBosonDisputeHandler.sol as well